src/app/core/local_account_sensitive_settings.nim

```diff
@@ -33,8 +33,11 @@ const DEFAULT_USE_COMPACT_MODE = true
 const LSS_KEY_TIMELINE_ENABLED* = "timelineEnabled"
 const DEFAULT_TIMELINE_ENABLED = true
 const LSS_KEY_RECENT_EMOJIS* = "recentEmojis"
+const DEFAULT_RECENT_EMOJIS = ""
 const LSS_KEY_HIDDEN_COMMUNITY_WELCOME_BANNERS* = "hiddenCommunityWelcomeBanners"
+const DEFAULT_HIDDEN_COMMUNITY_WELCOME_BANNERS = ""
 const LSS_KEY_HIDDEN_COMMUNITY_BACKUP_BANNERS* = "hiddenCommunityBackUpBanners"
+const DEFAULT_HIDDEN_COMMUNITY_BACKUP_BANNERS = ""
```

Comment:
@alexandraB99 Why are these constants empty? (P.S: sorry if my question is dumb)

Reply:
@borismelnik that's ok, because of integration QVariant in Nim, in order to have there anything but the Nim builtin types you should add appropriate QVariant constructor which will know how to handle that. But once it is exposed to qml then we're able to use it as you would use a regular QVariant on the qml side.

Since she needs Json arrays or objects as QVariant props this is an easier way do deal with them, but you have to check them on start on the qml side and convert them to json.

Reply:
thanks for explaining!